src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraCaptureSessionCallback.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Android.Hardware.Camera2;
+
+namespace BuildIt.Forms.Controls.Platforms.Android
+{
+    public class CameraCaptureSessionCallback : CameraCaptureSession.StateCallback
+    {
+        private readonly CameraPreviewControlRenderer owner;
+
+        public CameraCaptureSessionCallback(CameraPreviewControlRenderer owner)
+        {
+            if (owner == null)
+            {
+                throw new System.ArgumentNullException("owner");
+            }
+
+            this.owner = owner;
+        }
+
+        public override void OnConfigureFailed(CameraCaptureSession session)
+        {
```

Comment:
Do we need to do something sensible when this happens?

Reply:
added a todo to handle this in the next iteration so that a callback is exposed on the control